pkgs/top-level/perl-packages.nix

```diff
@@ -17611,8 +17611,8 @@ let
     pname = "PDF-API2";
     version = "2.042";
     src = fetchurl {
-      url = "mirror://cpan/authors/id/S/SS/SSIMMS/PDF-API2-2.038.tar.gz";
-      sha256 = "dEfEdJsCp4T1JdPH7OmdNLChBHXbZQlvYxZ0jdL5vQk=";
+      url = "mirror://cpan/authors/id/S/SS/SSIMMS/PDF-API2-2.042.tar.gz";
```

Comment:
Thanks for noticing this. We might want to switch to using `${version}` here, to make issues like this obvious.

Reply:
perlPackages has a strange trend torwards not using it.